aquarius/events/util.py

```diff
@@ -91,19 +92,33 @@ def get_address_file():
 
 
 def get_metadata_contract(web3):
-    """Returns a Contract built from the Metadata contract address and ABI"""
-    address_file = get_address_file()
-
-    with open(address_file) as f:
-        address_json = json.load(f)
-
-    network = get_network_name()
-    address = address_json[network]["Metadata"]
+    """Returns a Contract built from the Metadata contract address (or ENV) and ABI"""
+    address = os.getenv("METADATA_CONTRACT_ADDRESS", None)
```

Comment:
Isn't this contrary to what we are trying to achieve with the `artifacts` package? We worked towards removing hardcoding and assignments, why do we need to add that back? I personally edited these files recently, and I don't want us to counteract each other.

Reply:
I should be always allowed to overwrite what it's in artifacts:
 - testing new Metadata Contracts
 - deploy on unknown or new networks (maybe under NDA, so not allowed to have it our artifacts)

Reply:
@alexcos20 I disagree with the addition of the `METADATA_CONTRACT_ADDRESS` envvar.  It conflicts with the `ADDRESS_FILE` envvar.  If someone needs to override the Metadata contract address, they should just change the contents of the file pointed to by `ADDRESS_FILE`.